Types aren't compatible with React v18

It looks like the types might need to be updated for later versions of React. This is generated from a project using React v18.

```
$ tsc --noEmit
node_modules/react-countdown/dist/Countdown.d.ts:5:47 - error TS2694: Namespace 'React' has no exported member 'Props'.

5 export interface CountdownProps extends React.Props<Countdown>, CountdownTimeDeltaFormatOptions, Omit<LegacyCountdownProps, 'onComplete'> {
                                                ~~~~~


Found 1 error in node_modules/react-countdown/dist/Countdown.d.ts:5
```

I am seeing this as well when running:

``` 
$ tsc && vite build

node_modules/react-countdown/dist/Countdown.d.ts:5:47 - error TS2694: Namespace 'React' has no exported member 'Props'.

5 export interface CountdownProps extends React.Props<Countdown>, CountdownTimeDeltaFormatOptions, Omit<LegacyCountdownProps, 'onComplete'> {
                                                ~~~~~
Found 1 error in node_modules/react-countdown/dist/Countdown.d.ts:5
```

--- a/src/Countdown.tsx
+++ b/src/Countdown.tsx
@@ -12,8 +12,14 @@ import {
   formatTimeDelta,
 } from './utils';
 
+interface Props<T> {
+  children?: React.ReactNode;
+  key?: React.Key;
+  ref?: React.LegacyRef<T>;
+}
+
 export interface CountdownProps
-  extends React.Props<Countdown>,
+  extends Props<Countdown>,
     CountdownTimeDeltaFormatOptions,
     Omit<LegacyCountdownProps, 'onComplete'> {
   readonly date: Date | number | string;

